config/locales/es.yml

```diff
@@ -1448,110 +1449,110 @@ es:
         comparison:
           express:
-            description: "¡Archive rápidamente por su cuenta!"
-            timeframe: 15 minutos para presentar, disponible en marzo de 2022
-            title: Express
+            description: "¡Declare rápido por cuenta propia!"
+            timeframe: 15 minutos para declarar sus impuestos, disponible abril 2022
+            title: Express/Rápido
           gyr:
-            description: "¡Recoge el reembolso más grande! Presente con la ayuda de nuestros voluntarios certificados por el IRS."
-            timeframe: 2-3 semanas para presentar
+            description: "¡Cobre el reembolso más grande! Presente su declaración con asistencia de nuestros voluntarios certificados por IRS."
+            timeframe: De 2-3 semanas para declarar sus impuestos
             title: GetYourRefund
-        file_with_gyr: Archivo con GetYourRefund
+        file_with_gyr: Solicite por medio de GetYourRefund
         refund: Reembolso
-        sign_up_for_express: Suscríbete a Express
-        title: "¡Tenemos dos opciones gratuitas que pueden funcionar para usted!"
+        sign_up_for_express: Subscríbase para Express
+        title: "¡Tenemos dos opciones gratis que pueden funcionar para usted!"
     triage_gyr:
       edit:
-        file_in_person: Presentar en persona
-        file_on_my_own: En realidad, prefiero presentar la solicitud rápidamente por mi cuenta sin ayuda.
-        file_online: Archivo en línea
+        file_in_person: Presente su declaración en persona
+        file_on_my_own: 'En realidad, quiero intentar presentar mi declaración rápidamente por mi cuenta utilizando software gratuito. '
+        file_online: Presente su declaración en línea
         help_text: |
-          Recoja todos los créditos fiscales que le deben con asistencia.
+          Cobre todos los créditos tributarios que se le deben con asistencia.
 
           ¡Disponible en línea o en persona!
 
-          ¡Nuestro equipo de impuestos certificado por el IRS le brindará apoyo práctico con sus impuestos!
-        title: "¡Recomendamos presentar con nuestra opción GetYourRefund gratuita!"
+          ¡Nuestro equipo certificado por el IRS le brindará apoyo práctico con sus impuestos!
+        title: "¡Le recomendamos que presente su declaración con nuestra opción gratuita de GetYourRefund!"
     triage_id_type:
       edit:
         ssn_itin_type:
-          have_id: Tengo mi tarjeta de Seguro Social o papeleo ITIN
-          know_number: Sé mi número de Seguro Social o ITIN, pero no tengo una copia del mismo
-          need_help: Necesito ayuda para solicitar mi ITIN
+          have_id: Tengo mi tarjeta de Número de Seguro Social o la documentación de mi Número de Identificación Personal del Contribuyente o ITIN en inglés
+          know_number: 'Me sé mi Número de Seguro Social o Número de Identificación Personal del Contribuyente o ITIN en inglés, pero no tengo una copia '
```

Comment:
There is an unnecessary ` ` (space) character at the end of this string. It is not a problem, but it is unnecessary.